Makefile

```diff
@@ -52,47 +52,40 @@ clean-test: ## remove test and coverage artifacts
 lint: ## check style with flake8
 	pipenv run flake8 --ignore=E501 git_wrapper tests integration_tests
 
-test: dev ## run tests quickly with the default Python
-	pipenv run py.test tests
+test: ## run tests quickly with the default Python
```

Comment:
Do we need the Makefile at this point? If it's still providing value then we can keep it. I have a feeling very feel people will actually use it, especially if we have automation around building.

Reply:
I know I originally suggested we simply toss the Makefile, and yes, we still could do that if everyone wants.  However, while working on this patch, and trying to update the Makefile, I actually found it super handy, especially for being able to run 'make docs', and have it clean, build and launch in a browser, vs having to 'cd docs/;make clean; make html; <navigate to file in browser'.  Perhaps I should add the alternative direct pipenv (or even virtualenv, like in @jpichon 's other patch, which I hadn't even seen) directions to the README or CONTRIBUTING docs.

Reply:
Also, I think we should switch to having the make/virtualenv/pipenv directions only in one place, I didn't notice before they were duplicated.

Reply:
I wonder if we might just remove the parts we don't use and keep e.g. the docs part only, but it seems fine to punt that conversation, especially now that many of the references are to tox jobs and thus avoid some of the duplication.

Reply:
Yeah, personally, I would like to keep it, at least for now.  I know I am not a primary contributor to this codebase, but just in doing these packaging changes, I used it extensively to test setting up a fresh environment, running tests, building docs, etc.  Doing it that way actually exposed a number of errors I was able to correct in my changes.  There is also a convenience level to running `make tests` vs `pipenv run tox -epy`, though I know that is more subjective.

Reply:
This is fine by me if it's proven useful! And it might be now that it's up to date again.